stable/pipelines/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # JFrog Pipelines Chart Changelog
 All changes to this chart to be documented in this file
 
+## [1.5.7] September 30, 2020
```

Comment:
@rahulnlb1 Can you rebase your changes ?

Reply:
Done